sys-meta/cpio/cpio-0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="CPIO symlink"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="+gnu libarchive"
+REQUIRED_USE="^^ ( gnu libarchive )"
+
+RDEPEND="
+	gnu? ( >=app-arch/cpio-2.13-r4 )
+	libarchive? ( app-arch/libarchive )
+	!<app-arch/cpio-2.13-r4
+"
+
+src_install() {
+	if use gnu; then
+		dosym gcpio /bin/cpio
+		newman - cpio.1 <<-EOF
+			.so gcpio.1
+		EOF
+	elif use libarchive; then
+		dosym ../usr/bin/bsdcpio /bin/cpio
```

Comment:
Reminder merged-usr /usr/bin/cpio will be broken.

Reply:
Make a pull request to my pull request? I don't have a merged-usr setup to test.

Reply:
I don't run merged-usr either, but you can just get a merged usr stage3 that we have on the download page now (converting is quick too, emerge+run merge-usr, select merged-usr profile, update world).

merge-usr script can thankfully auto-correct links, aka `cpio -> ../usr/bin/bsdcpio` becomes `cpio -> bsdcpio`, but if emerge sys-meta/cpio again then it'll try to remake the link wrong without a `USE=split-usr` to handle that

Reply:
Ok, I think I've done that in a working way.